File: core-java/src/main/java/com/baeldung/datetime/UsePeriod.java
package com.baeldung.datetime;

import java.time.LocalDate;
import java.time.Period;

class UsePeriod {

    LocalDate modifyDates(LocalDate localDate, Period period) {
        return localDate.plus(period);
    }

    Period getDifferenceBetweenDates(LocalDate localDate1, LocalDate localDate2) {
        return Period.between(localDate1, localDate2);
    }
}

File: core-kotlin-modules/core-kotlin-date-time/src/main/kotlin/com/baeldung/dates/datetime/UsePeriod.kt
package com.baeldung.dates.datetime

import java.time.LocalDate
import java.time.Period

class UsePeriod {

    fun modifyDates(localDate: LocalDate, period: Period): LocalDate {
        return localDate.plus(period)
    }

    fun getDifferenceBetweenDates(localDate1: LocalDate, localDate2: LocalDate): Period {
        return Period.between(localDate1, localDate2)
    }
}